src/expr/src/relation/mod.rs

```diff
@@ -2058,8 +2056,7 @@ impl AggregateExpr {
             | AggregateFunc::MinTimestampTz
             | AggregateFunc::Any
             | AggregateFunc::All
-            | AggregateFunc::Dummy => self.expr.is_literal(),
-            AggregateFunc::Count => self.expr.is_literal_null(),
+            | AggregateFunc::Count => self.expr.is_literal_null(),
```

Comment:
Now all of them became `is_literal_null` instead of `is_literal`. I think this is causing the `max(2)` test regression, through the `is_constant` call in `LiteralLifting` line 556.

Reply:
Oh oops. I botched this. Thank you very much for noticing it!

Reply:
No problem!

Reply:
All fixed, and the test diffs are now gone. Thank you for the careful read there.